### PR TITLE
vbump deps

### PR DESCRIPTION
**Pull request by pawelru**
WIP

### CODE REVIEW

File: .github/workflows/integration-tests.yaml [hunk 1]
```diff
@@ -5,6 +5,9 @@ on:
   schedule:
     - cron: '45 3 * * 0'
   workflow_dispatch:
+  push:
+    branches:
+      - min_deps
```
Comment (pawelru) [inline .github/workflows/integration-tests.yaml:10]: Need to revert it once completed

```suggestion
```

File: .github/workflows/integration-tests.yaml [hunk 2]
```diff
@@ -5,6 +5,9 @@ on:
...
```
Comment (pawelru) [inline .github/workflows/integration-tests.yaml:11]:
```suggestion


```

File: .github/workflows/integration-tests.yaml [hunk 3]
```diff
@@ -5,6 +5,10 @@ on:
...
+
```
Comment (pawelru) [inline .github/workflows/integration-tests.yaml:11]:
```suggestion
```

**Review by gmbecker — CHANGES_REQUESTED** [review]
Do we have evidence that versions earlier than these cause problems? If not, this ends up inpractice imposing a pretty strict modernity requirement on rtables which is at odds with the way production systems tend to move and be updated slowly.

**Review by gmbecker — CHANGES_REQUESTED** [review]
Do we have evidence that versions earlier than these cause problems? If not, this ends up inpractice imposing a pretty strict modernity requirement on rtables which is at odds with the way production systems tend to move and be updated slowly.

**Pull request closed without merging**